integration-cli/docker_cli_service_logs_test.go

```diff
@@ -91,7 +91,7 @@ func (s *DockerSwarmSuite) TestServiceLogsCompleteness(c *testing.T) {
 	lines := strings.Split(strings.TrimSpace(out), "\n")
 
 	// i have heard anecdotal reports that logs may come back from the engine
-	// mis-ordered. if this tests fails, consider the possibility that that
+	// mis-ordered. if this tests fails, consider the possibility that
```

Comment:
I think this one was intentional 😅 I wonder if we should also ignore `that`, since `that that` can be grammatically correct.

Reply:
Yeah, that said (no pun); `that that` tends to be confusing in many case (that _what_ exactly?).

I was played it in my head, and thought  `consider the possibility that might be occurring` was grammatically correct here (and within this context, the equivalent to "that that might be occurring"), but maybe I'm GPT-hallucinating here; @dvdksn to the rescue?

Reply:
Right, imo both ultimately sound okay and neither is very vague in this case, but as I understand it the first `that` is a conjunction which introduces the other clause (`that might be occurring`), and the other `that` is a part of the clause being introduced `that (events are coming mis-ordered) might be occurring`.

I'm **sure** this block could be rewritten better without the double-that.

Reply:
lazy rephrasing

```suggestion
	// mis-ordered. this test failing may indicate that mis-ordering
```

Reply:
I rewrote it a bit;

```go
// I have heard anecdotal reports that logs may come back from the engine
// mis-ordered. If this tests fails, then possibly that's what causing the
// failure.
```

Also wondering if the "anecdotal reports" were related to this ticket;

- https://github.com/moby/moby/issues/31706 

I was considering adding a link to that, but service logs and container logs are implemented differently, so perhaps it's very unrelated.